src/components/AppRoute.jsx

```diff
@@ -3,16 +3,12 @@ import { Route, Redirect } from 'react-router'
 
 import App from './App'
 import Folder from '../containers/Folder'
-const ComingSoon = () => (<p style='margin-left: 2em'>Coming soon!</p>)
 
 const AppRoute = (
   <Route component={App}>
     <Redirect from='/' to='files' />
-    <Route path='files(/:file)' component={props => <Folder context='files' {...props} />} />
-    <Route path='recent' component={ComingSoon} />
-    <Route path='shared' component={ComingSoon} />
-    <Route path='activity' component={ComingSoon} />
-    <Route path='trash(/:file)' component={props => <Folder context='trash' {...props} />} />
+    <Route path='files(/:folderId)' component={props => <Folder context='files' {...props} />} />
```

Comment:
I agree `context` is not ideal, but honestly renaming it now would require a bit more work than I'd be inclined to ;) Also, I find `folderType`worse.